tools/Common/IceRpc.Telemetry.Internal/Program.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc;
+using IceRpc.Retry;
+using IceRpc.Telemetry.Internal;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+const int timeout = 3000; // The timeout for the RPC call in milliseconds.
+const int maxAttempts = 3; // The maximum number of attempts to retry the RPC call.
+const string uri = "icerpc://localhost"; // The URI of the server.
+
+// Load the root CA certificate
+using var rootCA = new X509Certificate2("certs/cacert.der");
+var clientAuthenticationOptions = new SslClientAuthenticationOptions
+{
+    RemoteCertificateValidationCallback = (sender, certificate, chain, errors) =>
+    {
+        using var customChain = new X509Chain();
+        customChain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
+        customChain.ChainPolicy.DisableCertificateDownloads = true;
+        customChain.ChainPolicy.TrustMode = X509ChainTrustMode.CustomRootTrust;
+        customChain.ChainPolicy.CustomTrustStore.Add(rootCA);
+        return customChain.Build((X509Certificate2)certificate!);
+    }
+};
```

Comment:
Once we have the telemetry server deployed, we can just use the default `SslClientAuthenticationOptions` and will not need to add a test cert to the Trust store.